WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -661,6 +672,49 @@ - (void)showUpdateSiteIconAlert
     [self presentViewController:updateIconAlertController animated:YES completion:nil];
 }
 
+- (void)uploadDroppedSiteIconImage:(UIImage *)image
```

Comment:
I'd probably rename this to `presentCropViewControllerForDroppedSiteIcon:` or something, as that's really what this method is doing.